src/python_testing/TC_DEM_2_2.py

```diff
@@ -23,7 +23,7 @@
 # test-runner-run/run1/app: ${ENERGY_MANAGEMENT_APP}
 # test-runner-run/run1/factoryreset: True
 # test-runner-run/run1/quiet: True
-# test-runner-run/run1/app-args: --discriminator 1234 --KVS kvs1 --trace-to json:${TRACE_APP}.json --enable-key 000102030405060708090a0b0c0d0e0f --featureSet 0x01
+# test-runner-run/run1/app-args: --discriminator 1234 --KVS kvs1 --trace-to json:${TRACE_APP}.json --enable-key 000102030405060708090a0b0c0d0e0f --featureSet 0x01 --application evse
```

Comment:
Why do we need to precise the app here if the app is already mentioned three lines above:
`${ENERGY_MANAGEMENT_APP}` ?

Reply:
The energy management app itself can run as a water heater or evse personality, so this tells the chip-energy-management-app to run with the EVSE cluster and delegates instantiated (rather than the water heater).

Reply:
The -application option can be either evse or water-heater. The energy management app can be run in one of two modes: EVSE or water-heater-management mode.

Reply:
Resolving so the PR can merge